docs/documentation/stories/global-scripts.md

```diff
@@ -32,12 +32,25 @@ global library, and one imported as a module.
 This is especially bad for libraries with plugins, like JQuery, because each copy will have
 different plugins.
 
-Instead, download typings for your library (`npm install @types/jquery`) which will give you
-access to the global variables exposed by that library.
+Instead, download typings for your library (`npm install @types/jquery`) and follow
+the [3rd Party Library Installation](https://github.com/angular/angular-cli/wiki/stories-third-party-lib) steps,
+this will give you access to the global variables exposed by that library.
 
 If the global library you need to use does not have global typings, you can also declare them
 manually in `src/typings.d.ts` as `any`:
 
 ```
 declare var libraryName: any;
-```
\ No newline at end of file
+```
+
+When working with scripts that extend other libraries like may be the case with JQuery plugins
```

Comment:
This sentence reads a bit weird with the `may` there, perhaps it should be `extend other libraries, for instance with with JQuery plugins`?